packages/trader/src/Modules/SmartChart/Components/bottom-widgets.tsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const BottomWidgets = ({ Widget }: { Widget: React.ReactNode }) => <div className='bottom-widgets'>{Widget}</div>;
```

Comment:
Tried with `React.PropsWithChildren` but didn't work for me

Reply:
it's ok because we don't use children here :)